configuration/controller-HA.cfg

```diff
@@ -45,44 +38,84 @@
 # If it is a relative path, then it is relative to the directory where this file
 # resides.
 
-POSTDIR="controller"
+POSTDIR="."
 
 # Array of required post scripts, in the order in which they will run.
 # NOTE: Those are basenames, without any extension -- not the actual name of the
 #       script files themselves.
 
-POSTLIST=( \
-            standard-configuration \
-            yum-cache \
-            hosts \
-            local-repos \
-            base-packages \
-            yum-update \
-            additional-repos \
-            additional-packages \
-            firewalld \
-            fail2ban \
-            chrony \
-            environment-modules \
-            applications \
-            rdma-centos \
-            nfs-server \
-            openldap \
-            sssd \
-            bind \
-            mariadb \
-            slurm-pre \
-            slurm \
-            luna \
-            postfix \
-            zabbix \
-            # docker-registry \
-            # docker \
-            zabbix-agent \
-            services-cleanup \
+POSTLIST=(
+            # HA-independant base packages
+            # Those do not require trinity.sh, the shared FS or Pacemaker
+            controller/yum-cache
+            controller/local-repos
+            controller/base-packages
+            controller/additional-repos
+            controller/additional-packages
+            controller/yum-update
+            controller/hosts
+            controller/firewalld
+            controller/chrony
+            controller/rdma-centos
+
+            # Critical section begins here ------------------
+            # Do not change the order of those post scripts!
+            +STEP
+            controller/detect-primary
+            controller/critical-section-beginning
+            controller/ssh
+            controller/pacemaker
+            filesystem/shared-storage
```

Comment:
Shared storage might not be available during the install of the first controller. It might be shared storage which requires configuration: Dell PowerVault for instance. The same valid for NFS server or any external shared resources. First controller is literally The First. The rest of infrastructure is not ready.